network/p2p/p2p_setup.go

```diff
@@ -48,6 +47,9 @@ const (
 	connectTimeout = time.Minute
 	// connectorQueueSize is the buffer size of the channel used by the connector
 	connectorQueueSize = 256
+	// inboundLimitRatio is the ratio of inbound connections to the total connections
+	// we allow (both inbound and outbound).
+	inboundLimitRatio = float64(0.75)
```

Comment:
I'm testing now if `0.75` is a good fit, but so far
- I found the value of `0.5` to be working well
- and the value of `0.8` seems to be noticeably worse (eg. making it 30 minutes instead of 15 to get rid of dead subnets)

Reply:
I guess `0.75` is too high still, even with MaxPeers = 90 sometimes it can take up to 90 minutes to get rid of dead subnets (or even longer, if we are unlucky enough),

it's probably better to set `inboundLimitRatio` = 0.6 or even lower